docs/rst/tutorials/core/qos/ownership/ownership.rst

```diff
@@ -19,7 +19,7 @@ This QoS Policy specifies whether it is allowed for multiple DataWriters to upda
 Prerequisites
 -------------
 
-The first prerequisite is to have Vulcanexus Humble installed (see `Linux binary installation <https://docs.vulcanexus.org/en/latest/rst/installation/linux_binary_installation.html>`_ or `Linux instalation from sources <https://docs.vulcanexus.org/en/latest/rst/installation/linux_source_installation.html>`_)
+The first prerequisite is to have Vulcanexus Humble installed (see `Linux binary installation <https://docs.vulcanexus.org/en/latest/rst/installation/linux_binary_installation.html>`_ or `Linux installation from sources <https://docs.vulcanexus.org/en/latest/rst/installation/linux_source_installation.html>`_)
```

Comment:
Same comment as before. If kept, I suggest specifying the metapackage required for this specific tutorial

Reply:
Should this PR modify that tutorial? IDK

Reply:
Hahahaha It has already modified the spelling, but it can be done in another PR or leave as it is. It is not wrong per se. It is only a suggestion to keep some consistency between the tutorials. But being contributed by different authors is only logical that some differences exist.